### DISCUSSION TITLE
More energy added than used - is this normal?

### DISCUSSION THREAD

**Discussion by woyteck1** [teslamate-org/teslamate#2609] [General]
Hi,
Recently I have noticed that my car is reporting more energy added than used making the charging efficiency over 100%, whereas before it usually was less added than used thus sometimes the charging efficiency (especially on slow charges and on Superchargers with battery heating) was less than 100%.
An example screenshot:
![image](https://user-images.githubusercontent.com/25311523/163987331-35227d09-ff6d-41aa-9415-32cc6a0ad445.png)

Any idea what would cause this?

**Comment by ktsenvolt** [comment]
![image](https://user-images.githubusercontent.com/80360674/164132284-5bada30a-ae5d-4bfd-b6e1-c9382baf2377.png)
It happened to me as well sometime but not always.
